host/windows/syscall.c

```diff
@@ -3032,3 +3032,69 @@ int oe_syscall_nanosleep_ocall(struct oe_timespec* req, struct oe_timespec* rem)
     _set_errno(0);
     return 0;
 }
+
+/*
+**==============================================================================
+**
+** clock_gettime():
+**
+**==============================================================================
+*/
+
+/*
+**==============================================================================
+**
+** POSIX_TO_WINDOWS_EPOCH_TICKS:
+**
+** The ticks elapsed since 1970-01-01 00:00 (UTC). This value was derived with
+** the following function.
+**
+**     LONGLONG get_posix_to_windows_epoch_ticks()
+**     {
+**         SYSTEMTIME st = { .wYear = 1970, .wMonth = 1, .wDay = 1 };
+**         FILETIME ft;
+**         ULARGE_INTEGER x;
+**
+**         SystemTimeToFileTime(&st, &ft);
+**         x.u.LowPart = ft.dwLowDateTime;
+**         x.u.HighPart = ft.dwHighDateTime;
+**
+**         return x.QuadPart;
+**     }
+**
+**==============================================================================
+*/
+
+const LONGLONG POSIX_TO_WINDOWS_EPOCH_TICKS = 0X19DB1DED53E8000;
+
+/* Return milliseconds elapsed since the Epoch. */
```

Comment:
The comment does not reflect what the function does ie populate the fields of oe_timespec parameter.